crates/bevy_ecs/src/reflect.rs

```diff
@@ -418,15 +418,30 @@ impl ReflectMapEntities {
         world: &mut World,
         entity_map: &EntityMap,
     ) -> Result<(), MapEntitiesError> {
-        (self.map_entities)(world, entity_map)
+        (self.map_entities)(world, entity_map, entity_map.values().collect())
+    }
+
+    /// This is like `map_entities`, but only applied to specific entities, not all values in the `EntityMap`.
+    ///
+    /// This is useful for when you only want to apply the `MapEntity` logic to specific entities. For example,
+    /// a scene can be loaded with `Parent` components, but then a `Parent` component can be added to these entities
```

Comment:
Nit: I wonder if we should avoid mentioning "scenes" in `bevy_ecs` content like this since it's not actually a dependency of the crate. Perhaps we use the concept of "worlds" since scenes are basically a tiny and limited `World`?

@alice-i-cecile probably has more knowledge in this area

Reply:
I think that this is probably net useful: humans can figure this out.